test/runtime/samples/module-context-bind/main.svelte

```diff
@@ -0,0 +1,11 @@
+<script context='module'>
+	let foo;
+</script>
+
+<script>
+	import { onMount } from 'svelte';
+	let bar;
+	onMount(() => bar = foo);
```

Comment:
If there's a nicer way to find out in the test what the value of the module-level `foo` variable is once the component's mounted, I couldn't find it.